src/lib/swankyCommand.ts

```diff
@@ -40,68 +40,86 @@ export abstract class SwankyCommand<T extends typeof Command> extends Command {
       args: this.ctor.args,
       strict: this.ctor.strict,
     });
+
     this.flags = flags as Flags<T>;
     this.args = args as Args<T>;
 
     this.logger = swankyLogger;
     this.swankyConfig = buildSwankyConfig();
-    try {
-      const systemConfig = await getSwankySystemConfig();
 
-      Object.entries(systemConfig).forEach((entry) => {
-        this.swankyConfig[entry[0] as keyof SwankyConfig] = entry[1];
-      });
+    await this.loadAndMergeConfig();
+
+    this.logger.info(`Running command: ${this.ctor.name}
+      Args: ${JSON.stringify(this.args)}
+      Flags: ${JSON.stringify(this.flags)}
+      Full command: ${JSON.stringify(process.argv)}`);
+  }
+
+  protected async loadAndMergeConfig(): Promise<void> {
+    try {
+      const systemConfig = getSwankyConfig("global");
+      this.swankyConfig = { ...this.swankyConfig, ...systemConfig };
     } catch (error) {
-      await this.storeConfig(this.swankyConfig, 'global');
+      this.warn(
+        `No Swanky system config found; creating one in "/${DEFAULT_CONFIG_FOLDER_NAME}/${DEFAULT_CONFIG_NAME}}" at home directory`
+      );
+      await this.storeConfig(this.swankyConfig, "global");
     }
 
     try {
-      const localConfig = await getSwankyConfig();
-
-      Object.entries(localConfig).forEach((entry) => {
-        this.swankyConfig[entry[0] as keyof SwankyConfig] = entry[1];
-      });
+      const localConfig = getSwankyConfig("local") as SwankyConfig;
+      this.mergeAccountsWithExistingConfig(this.swankyConfig, localConfig);
```

Comment:
We should be careful when we merge accounts because after the merge we don't know where an account is coming from. Every time we store the config, we should check which accounts are local or global if we want to leave this merging.